include/dem/insertion_file.h

```diff
@@ -24,19 +24,19 @@
 
 #include <deal.II/particles/particle_handler.h>
 
-#ifndef lethe_file_insertion_h
-#  define lethe_file_insertion_h
+#ifndef lethe_insertion_file_h
+#  define lethe_insertion_file_h
 
 template <int dim>
-class FileInsertion : public Insertion<dim>
+class InsertionFile : public Insertion<dim>
 {
 public:
-  FileInsertion(const DEMSolverParameters<dim> &dem_parameters,
+  InsertionFile(const DEMSolverParameters<dim> &dem_parameters,
                 const std::vector<std::shared_ptr<Distribution>>
                   &distribution_object_container);
 
   /**
-   * @brief The FileInsertion class inserts particles using data stored in a file.
+   * @brief The InsertionFile class inserts particles using data stored in a file.
```

Comment:
Not specific for this PR, but we should avoid to write "The ClassName class do..." for the Doxygen doc. If you plan to apply some comment before merge of this PR, I would remove it (so just "Insert particles..."), so it will be done. :)